pkg/query-service/app/http_handler.go

```diff
@@ -1397,8 +1397,11 @@ func (aH *APIHandler) submitFeedback(w http.ResponseWriter, r *http.Request) {
 		"email":   email,
 		"message": message,
 	}
-	telemetry.GetInstance().SendEvent(telemetry.TELEMETRY_EVENT_INPRODUCT_FEEDBACK, data)
-
+	ctx := auth.AttachJwtToContext(r.Context(), r)
+	userEmail, err := auth.GetEmailFromJwt(ctx)
```

Comment:
can make a one liner for this, since every where two lines are getting replicated ?

Reply:
These two lines are not replicated everywhere and these two lines perform separate functions. So creating a function doesn't makes sense.

Reply:
Moved AttachJwtToContext to middleware